i18n/nl.json

```diff
@@ -14,10 +14,10 @@
     "bounces.unknownService": "Onbekende service.",
     "bounces.view": "Zie bounces",
     "campaigns.addAltText": "Voeg alternatieve tekst zonder opmaak toe",
-    "campaigns.archive": "Archive",
-    "campaigns.archiveEnable": "Publish to public archive",
-    "campaigns.archiveHelp": "Publish (running, paused, finished) the campaign message on the public archive.",
-    "campaigns.archiveMeta": "Campaign metadata",
+    "campaigns.archive": "Archiveren",
```

Comment:
Sorry for the separate review, but I think this should be the noun archive ('Archief'), not the verb ('Archiveren'). At least that's how it is in the german translations. @knadh?

Reply:
Yep, it should be the noun.

Reply:
@TychoWerner Could you please amend the PR with this change?

Reply:
These changes are valid.